content/10.zk-stack/20.running-a-zk-chain/30.custom-base-tokens.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Custom Base Tokens
+description:
+---
+
+You can customize your ZK chain by changing the base token used for gas from ETH to an ERC20 token.
+
+With the initial release of ZK stack, base tokens must be whitelisted for use as a base token for a chain.
```

Comment:
```suggestion
To use a token as a base token on a ZK chain, it must be whitelisted in the `BridgeHub` contract on the L1.
```
A base token is only a base token if it is whitelisted, even if it is ETH? Is ETH the default base token and it doesn't need to be whitelisted, or is this a process for any ERC20 token that is going to be a base token?

Reply:
it's just for ERC20 tokens afaik